snyk_scm_refresh.py

```diff
@@ -206,8 +206,8 @@ def import_github_repo(org_id, owner, name):
     try:
         response = org.client.post(path, payload)
     except snyk.errors.SnykHTTPError as err:
-        if err.code == 502:
-            print("We've been asked to wait, lets try again in a minute...")
+        if err.code in [502,504]:
+            print("Server error, lets try again in a minute...")
             time.sleep(60)
             response = org.client.post(path, payload)
```

Comment:
```suggestion
           try:
               response = org.client.post(path, payload)
           except snyk.errors.SnykHTTPError as e:
               print(f"Still failed after retry with {str(e.code)}!")
               raise           
```

Reply:
Or alternatively, we could make our own exception that encapsulates the various Snyk issues and raise it here instead of reraising the Snyk exceptions.